src/test/java/io/jenkins/plugins/pipelinegraphview/PipelineGraphViewCancelTest.java

```diff
@@ -37,17 +35,15 @@ void cancelButtonCancelsBuild(Page p, JenkinsConfiguredWithCodeRule j) throws Ex
                 .goToBuild()
                 .goToPipelineOverview();
 
-        assertTrue(p.getByRole(AriaRole.BUTTON, new Page.GetByRoleOptions().setName("Cancel"))
-                .isVisible());
+        Locator cancelLocator = p.getByRole(AriaRole.BUTTON, new Page.GetByRoleOptions().setName("Cancel"));
+        assertThat(cancelLocator).isVisible();
```

Comment:
There doesn't seem to be any waiting here, (except the one wait for hidden that is removed), could you explain this a bit? the `isVisible` and `isHidden` checks you've added won't wait for them to be true as far as I can tell?

Reply:
Ah, `Time to retry the assertion for in milliseconds. Defaults to 5000.`